polyfills/Array/from/polyfill.js

```diff
@@ -1,3 +1,44 @@
+
+function parseIterable(arraylike) {
```

Comment:
I think this is better off in a closure as to avoid other polyfills getting access to the function